proto/cmp/types/v1alpha1/product_list.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package cmp.types.v1alpha1;
+
+// Product list code and type
+//
+// This is being used in requests to specify the a list of products to be included in the reponse
+//
+// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha1/product-list.proto.dot.xs.svg)
+// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha1/product-list.proto.dot.svg)
+message ProductList {
+  string product_code = 1;
+  ProductCodeType product_code_type = 2;
+}
```

Comment:
In my opinion, this message type's naming is misleading. It suggests an already plural type Product**List**, but the actual message type is not. I suggest we change the naming to reflect that

```suggestion
message ProductCode {
  string code = 1;
  ProductCodeType type = 2;
}
```

Reply:
Agreed and implemented